website_sale_product_brand/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# © 2016 Serpent Consulting Services Pvt. Ltd. (http://www.serpentcs.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Brand Filtering in Website',
+    'category': 'e-commerce',
+    'author': "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
```

Comment:
@JayVora-SerpentCS @pedrobaeza we are maintaining this module from v9 with fix + improvement + migrating.  Do you think is fair to add here Tecnativa o not?

Reply:
Fine to add @rafaelbn ! Thanks.